server/api/reports.go

```diff
@@ -274,15 +271,16 @@ func independentVars(r TestReport) map[string]string {
 	return map[string]string{
 		// r.esHost() is an independent variable, but not queryable by the user
 		// esHost() is always an implicit filter for each query
-		"duration":    r.Duration.String(),
-		"events":      strconv.Itoa(r.Events),
-		"spans":       strconv.Itoa(r.Spans),
-		"frames":      strconv.Itoa(r.Frames),
-		"concurrency": strconv.Itoa(r.Concurrency),
-		"revision":    r.Revision,
-		"branch":      r.Branch,
-		"apm_host":    r.ApmHost,
-		"limit":       strconv.Itoa(int(r.Limit)),
+		"duration":     r.Duration.String(),
+		"errors":       strconv.Itoa(r.Errors),
```

Comment:
fyi: this change is why stress tests work and you can query for `errors` and/or `transactions` instead of `events`